alpine/2.3.6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Only alpine version that suports rethinkdb is edge
+FROM alpine:edge
```

Comment:
If there's no stable version of Alpine that supports RethinkDB, then we probably shouldn't be packaging for it yet (at least, not as the new base for the official image). The few Dockerfiles in the official library that are packaged for Alpine (it would seem that [the announcement of increased forward adoption of Alpine a year and a half ago](https://www.brianchristner.io/docker-is-moving-to-alpine-linux/) has been greatly exaggerated) don't do it without pinning a specific release, and `edge` is considered [unsuitable for production](https://wiki.alpinelinux.org/wiki/Edge).

Alternately, this could be listed as a Dockerfile *for Alpine Edge*, but it would need to go in a *subdirectory* for Alpine release versions (just as the repo is currently divided into separate directories for different Debian and Ubuntu releases). If doing this, I'd introduce a top-level distro directory level to the hierarchy (ie. moving the repo's `jessie` directory to `debian/jessie`) - I skipped this when originally creating the repo because, at the time I originally devised the current hierarchy, the intersection of "distros with official Docker images" and "distros that RethinkDB upstream builds packages for" was exclusively limited to the Debian / Ubuntu non-conflicting-named-release family.

Reply:
Actually Alpine 3.7 stable ships RethinkDB, no need for edge.

Reply:
As I noted [26 days ago](https://github.com/rethinkdb/rethinkdb-dockerfiles/pull/40#issuecomment-350838227), Alpine's actually been packaging RethinkDB since 3.5.